src/lightning_app/utilities/app_logs.py

```diff
@@ -78,8 +94,8 @@ def _app_logs_reader(
             project_id=project_id,
             app_id=app_id,
             component=component_name,
-            on_message_callback=_push_logevents_to_read_queue_callback(component_name, read_queue),
-            on_error_callback=_error_callback,
+            on_message_callback=_push_log_events_to_read_queue_callback(component_name, read_queue),
+            on_error_callback=on_error_callback or _error_callback,
```

Comment:
`_error_callback` here is only printing to stdout, should we instead log to stderr at least?

Reply:
It is printing to stderr.

Reply:
```
print(f"Error while reading logs ({errors.get(type(error), 'Unknown')})", file=sys.stderr)
```

Reply:
Argh, my bad